kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/AddFullyQualifiedNameController.java

```diff
@@ -114,8 +114,8 @@ public void initialize() {
      * This copies form values into the ViewModel's property values. It does not save or validate.
      */
     private void copyUIToViewModelProperties() {
-        if (fqnViewModel != null) {
-            fqnViewModel.setPropertyValue(NAME_TEXT, fullyQualifiedNameTextField.getText())
+        if (descrNameViewModel != null) {
```

Comment:
Can you remove the null check. Cognitive library injects the view model so it'll never be null. I know this was like that before, but I don't think we agreed to refactor all controllers. But since we are in here it can be easily fixed.